Makefile

```diff
@@ -117,13 +125,21 @@ test-fmt:
 	git diff --exit-code || (git --no-pager diff && git checkout . && exit 1)
 .PHONY: test-fmt
 
-lint: get-libs
+lint: get-libs $(GOLINES)
 	./bin/golangci-lint run --config .golangci.yml
+	@output="$$($(GOLINES) -m 120 --dry-run --base-formatter=gofmt --ignore-generated .)"; \
+	if [ -n "$$output" ]; then \
+		echo "golines found formatting issues:"; \
+		echo "$$output"; \
+		exit 1; \
+	fi
+
 .PHONY: lint
 
-# Auto-fixes golangci-lint issues where possible.
-lint-fix: get-libs
+# Auto-fixes golangci-lint and golines issues where possible.
+lint-fix: get-libs $(GOLINES)
 	./bin/golangci-lint run --config .golangci.yml --fix
+	$(GOLINES) -m 120 -w --base-formatter=gofmt --ignore-generated .
```

Comment:
If you want to keep this you have to execute it before `golangci-lint`. `golangci-lint` will detect lines longer than 120 characters, not fix them and then exit with 1 causing golines not to be executed.